lib/shopify-cli/context.rb

```diff
@@ -130,6 +130,38 @@ def write(fname, content)
       File.write(ctx_path(fname), content)
     end
 
+    # will change directories and update the root, the filepath is relative to the command root unless absolute
+    #
+    # #### Parameters
+    # * `path` - the file path to a directory, relative to the context root to remove from the FS
+    #
+    def chdir(path)
+      Dir.chdir(ctx_path(path))
```

Comment:
I actually wonder if we even need this line 🤔If all of our code uses the context methods, then it should all be relative to the root anyway, so line 140 might be all we need

Reply:
I'm a little confused because if we remove this then I don't think there is any method in context that actually changes the directory even if all of our code uses context.

Reply:
I'm a little confused because if we remove this then I don't think there is any method in context that actually changes the directory even if all of our code uses context.